packages/compass-app-stores/src/provider.ts

```diff
@@ -3,7 +3,7 @@ import { createContext, useContext } from 'react';
 
 export const InstanceContext = createContext<MongoDBInstance | null>(null);
 
-export const useMongoDBInstance = (): MongoDBInstance => {
+export const mongoDBInstanceLocator = (): MongoDBInstance => {
```

Comment:
@addaleax don't know if you have a strong preference here, I thought that I'd rename it to match other methods we have and to indicate a bit better that it's not supposed to be used as a hook in React (even though it sorta is).

I think if we want to add actual hooks here (should be very valuable in some cases!), those should probably be a bit more high level, encapsulating the model subscription logic for a required field internally instead of just exposing the modal to the React view 🤔

Reply:
sure, no strong preferences here! happy to stick to whatever pattern we choose :slightly_smiling_face: